cli/util/options.js

```diff
@@ -53,6 +54,7 @@ function parse(argv, config) {
       else { arguments.push(arg); continue; } // argument
     }
     if (option) {
+      provided.add(key);
```

Comment:
Isn't the contents of `provided` equivalent to `Object.keys(options)`?

Reply:
Oops. I put it in the wrong place, I think. Let me check.

Reply:
It should go into the boolean block, and in the nested if statement below. There.